services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -506,7 +506,7 @@ export const StudentViewContainer = ({ dispatch, session, isTurk, location, acti
 
   const className = `activity-container ${showFocusState ? '' : 'hide-focus-outline'} ${activeStep === READ_PASSAGE_STEP ? 'on-read-passage' : ''}`
 
-  if(!explanationSlidesCompleted) {
+  if(!explanationSlidesCompleted && (!activityCompletionCount || (activityCompletionCount && activityCompletionCount < 4))) {
```

Comment:
What if we just move this extra conditional up to line 75 where we set `explanationSlidesCompleted`.  That keeps the skip logic concentrated up near the key variables rather than spreading it out down here.  Or is that actually more confusing?

Reply:
If we give `activityCompletionCount` a default value, as suggested above, this line can just be `if (!explanationSlidesCompleted && activityCompletionCount > 3)`. Also, I think we should make whatever the minimum number of activities completed to skip the onboarding slide a constant at the top of the file for ease of changing - something like `ACTIVITY_COMPLETION_MAXIMUM_FOR_ONBOARDING`.

Reply:
good suggestion!